src/app/containers/Timestamp/helpers/timestampGenerator.js

```diff
@@ -0,0 +1,16 @@
+export default function timestampGenerator(timeDifference) {
```

Comment:
We generally follow the pattern
`const timestampGenerator = (timeDifference) => {`
and then at the end of the file
`export default timestampGenerator;`